packages/pds/src/db/database-schema.ts

```diff
@@ -25,8 +25,9 @@ import * as messageQueue from './tables/message-queue'
 import * as messageQueueCursor from './tables/message-queue-cursor'
 import * as sceneMemberCount from './tables/scene-member-count'
 import * as sceneVotesOnPost from './tables/scene-votes-on-post'
+import { Kysely } from 'kysely'
```

Comment:
A nit (you're welcome to ignore ✨ )— in the pds external dependencies seem to generally be imported above local dependencies by convention.